custom/ipi/variables.ipi.tmpl

```diff
@@ -48,6 +48,6 @@
   "Platform" => "Claim Category",
   "HardwarePlatform" => "Claim Category",
   "OperatingSystem" => "Source",
-  "url_to_documentation" => "http://youneedtotakecareofthat.com/"
+  "url_to_documentation" => "http://unee-t.com/"
```

Comment:
Should be `https://`

Reply:
Agreed, it should be https://
modified...